worldengine/generation.py

```diff
@@ -212,7 +213,7 @@ def generate_world(w, step):
 
     # FIXME: create setters
     IrrigationSimulation().execute(w, seed)
-    TemperatureSimulation().execute(w, seed)
+    
```

Comment:
It doesn't change anything just yet, but it will be changing things later on. Currently we are modifying the amount of rainfall based on the latitude. This is because areas with lower temperatures tend to experience lower rainfall (on average. There's lots of other weather effects as well but its not a bad starting rule and we are already using it, simply indirectly). To modify rainfall so it considers temperature means we have to move TemperatureSimulation to be triggered before PrecipitationSimulation.